source/enumerate-collections.rst

```diff
@@ -333,7 +331,7 @@ Example return (in PHP, but abbreviated)::
       [5] => class MongoCollection#11 { }
     }
 
-Drivers SHOULD specify the nameOnly option when executing the ``listCollections`` command for this method.
+Drivers MUST specify the ``nameOnly`` option when executing the ``listCollections`` command for this method.
```

Comment:
This seems to conflict with "SHOULD always specify the `nameOnly` option" from the "Getting Collection Names" section. Should we also use `MUST` there to ensure these are consistent?

That would leave "Getting Full Collection Information" as the lone differentiator, for which you already say that drivers MAY expose the option.